config/game_options.txt

```diff
@@ -122,6 +122,12 @@ PROTECT_ROLES_FROM_ANTAGONIST
 ## If non-human species are barred from joining as a head of staff, including jobs flagged as allowed for non-humans, ie. Quartermaster.
 #ENFORCE_HUMAN_AUTHORITY_ON_EVERYONE
 
+## If non-human species have their pay multiplied by the value of NONHUMAN_PAYDAY_MUL.
+#ENFORCE_NONHUMAN_WAGE_GAP
+
+## Multiplier for non-humans' paycheck values. Only used if ENFORCE_NONHUMAN_WAGE_GAP is uncommented.
+NONHUMAN_PAYDAY_MUL 0.75
+
```

Comment:
A) I believe the standing thought process for making previously non-configurable options configurable is to have the "base" config option match the previous code behavior. Since every non-human would always have their pay docked, that behavior should persist despite it becoming configurable (since server operators needing to both be aware of this and to update their config accordingly in order to achieve desired results (or at least results that they were expecting) is a bit silly). Should be something like:

```txt
## If non-human species get their pay docked. Uncommenting this variable will mean non-human species' pay will not be multiplied by the value of NONHUMAN_PAYDAY_MUL

#NO_NONHUMAN_WAGE_GAP
```

...and have the underlying code framework ensure that the switch must be toggled on the server's side, rather than try and "circumvent" the potential desires of a server operator. It's more of a "legacy" thing than anything, but it seems affable to me that I have to mirror the config just to get the "old behavior" back (let alone policy thread bullshit). 

That's how I see it (and I believe that's how I've been told to handle config changes such as these in the past).

B) There's no need to abbreviate `NONHUMAN_PAYDAY_MUL` imo. Just spell it out, we have the space for it.

Reply:
Good ideas, honestly. I was basing my choices off the fact that human supremacy defaults to _off_ even though I'm fairly certain it's enabled on tgstation servers, but maintaining current behavior seems good.